### DISCUSSION TITLE
How to use Android PackageInstaller with .NET MAUI

### DISCUSSION THREAD

**Discussion by Cruik** [dotnet/maui#14287] [Q&A]
Hey guys,

i try to update my own apk from within my app.
I found the PackageInstaller class for Android in the Android Documenation
https://developer.android.com/reference/android/content/pm/PackageInstaller

Does anybody have an code example on how to use it?
i am only concerning about the Android platform. iOs and windows does not need to be included here :)

**Comment by jlinker7** [comment]
I have code from Xamarin that I expected to continue to work in .net Maui.  But for some reason, it doesn't throw any errors, but when I commit the installer session it does nothing.  I see a message in logcat indicating the intent was sent, but nothing ever happens after that.  No errors, the apk is not installed, logcat shows nothing.  Code is identical to what it was in Xamain.
